action/protocol/staking/candidate_statemanager.go

```diff
@@ -14,20 +14,14 @@ import (
 	"github.com/iotexproject/iotex-address/address"
 
 	"github.com/iotexproject/iotex-core/action/protocol"
-	"github.com/iotexproject/iotex-core/state"
 )
 
 type (
-	// candidateBucketCenter contains candidate center and bucket pool
-	candidateBucketCenter interface {
-		// TODO: remove CandidateCenter interface, return *candCenter
-		CandCenter() CandidateCenter
-		BucketPool() *BucketPool
-	}
-
 	// CandidateStateManager is candidate state manager on top of StateManager
 	CandidateStateManager interface {
 		protocol.StateManager
+		CandCenter() CandidateCenter
+		BucketPool() *BucketPool
```

Comment:
If we do have these two APIs, we may delete all the following APIs, except "Commit"

Reply:
yes, or we can delete these 2 API